Rectangle.js

```diff
@@ -0,0 +1,14 @@
+function Rectangle (w, h) {
+  this.width = w
+  this.height = h
+  this.type = 'Rectangle'
+  this.description = function () {
```

Comment:
Method is best defined in the prototype so that you don't create one for each instance.